docs/general/polkadot-v1.md

```diff
@@ -0,0 +1,152 @@
+intermediaries or centralized entities.
+
+Currently, Polkadot 1.0 achieve resilience through the following strategies:
+
+- Nominated Proof of Staking (NPoS) where the stake per validator is maximized and evenly
+  distributed across validators.
+- The [1KV program](./thousand-validators.md) aims to incentivize new operators to become network
+  participants and further increase physical (how many validator nodes per service provider) and
+  social decentralization (how many validator nodes per operator). Those can be explored with the
+  [Polkawatch App](https://polkadot.polkawatch.app/).
+- An on-chain treasury and governance (see: [OpenGov](../learn/learn-polkadot-opengov.md) where every decision goes though public referenda and any
+  token holder can cast a vote.
+
+### Polkadot's Blockspace
+
+The design and realization of Polkadot 1.0 allowed its creators to enable commoditization of blockspace.
+
+A blockchain is a way to store data. The storage unit is the block, and once a block is finalized
+onto the chain, it is practically impossible to modify the data within that block. In addition to
+being tamper-proof, public permissionless blockchains like Polkadot store data that are visible to
+everybody (i.e. public), and anybody can become a network participant permissionlessly.
+
+**Blockspace** is the capacity of a blockchain to finalize and commit operations. It represents a 
+blockchain's security, computing, and storage capability as an end product. Blockspace produced by 
+different blockchains can vary in security, flexibility, and availability. 
+
+- **Security**, intended as how secure the blockspace is. In Proof-of-Stake (PoS) networks, this is
+  directly related to how much stake is locked on validator nodes, how much variance in stake there
+  is between validators (i.e. how easy it is to attack a single validator), and how many
+  validators there are securing the network (i.e. how easy it is for colluding validators to exert
+  influence on the network). Additionally, it is also important to look at how many validators are owned by single
+  operators (this will determine the degree of social centralization of the network), and how many
+  validators run on the same service provider (this will determine the degree of physical
+  centralization of the network).
+- **Flexibility**, intended as how flexible the blockspace is, what can be done with it, and what
+  type of data can be stored. Data quality plays an important role depending on the type of network.
+  One might avoid having situations in which poor quality data flood blockspace hindering the prompt
+  execution of vital processes.
+- **Availability**, intended as how available blockspace is and how difficult it is to access it. It
+  should not be too difficult to get your hands on it so that any business model (not only DeFi) can
+  thrive using it. Ideally, a marketplace must drive the blockspace price based on demand, with
```

Comment:
```suggestion
  should not be too difficult to get your hands on it so that any business model  can
  thrive using it. Ideally, a marketplace must drive the blockspace price based on demand, with
```